### PR TITLE
IG and QS script updates post deploy with LAC

### PR DESCRIPTION
**Pull request by DanPaseltiner**
This PR does the following:

1. Fixes the IG tables of contents
2. Replace "extract pipeline" with "tabulation service" to reflect the way we are currently describing out product.
3. Updates the requirements for a user to successfully run the QS script in the IG and intro text of the script itself.

### CODE REVIEW

File: docs/implementation-guide.md [hunk 1]
```diff
@@ -55,13 +51,12 @@ We have only provided a brief overview of PHDI, Building Blocks, and the pipelin
...
 
-### User Assumptions
-In order to proceed you will need either:
-- an Azure account with permissions to create new resource groups in your organization's Azure environment,
-or
-- an Azure account with `Owner` access to a resource group in your organizations’ Azure environment that was created to house the PHDI Azure pipelines, and the name of this resource group.
+### User Requirements
+In order to proceed you will need:
+1. `Owner` access to the Azure subscription where you would like to deploy the PHDI Starter Kit.
+2. `Admin` access to the GitHub account or organization where your copy of this repository will be created.
```
Comment (nickclyde) [inline docs/implementation-guide.md:57]: I'm not sure this is true - I think you just need enough permissions to be able to create repositories in the organization.

File: quick-start.sh [hunk 2]
```diff
@@ -50,8 +50,10 @@ fi
 
 # Intro text
 box "Welcome to the $(pink 'PHDI Azure') setup script!"
-echo "This script will help you setup $(pink 'Azure') authentication for GitHub Actions."
-echo "We need some info from you to get started."
+echo "This script will help you setup the PHDI Starter Kit in $(pink 'Azure'), including authentication for GitHub Actions."
+echo "In order for the script to work properly you will need:"
+echo "  1. Owner access to the Azure subscription where you would like to deploy the PHDI Starter Kit."
+echo "  2. Admin access to the GitHub account or organization where your copy of this repository will be created."
```
Comment (nickclyde) [inline quick-start.sh:56]: Same as comment above

**Review by nickclyde — COMMENTED** [review]
Looks good except for one comment:

File: docs/implementation-guide.md (hunk 1 above)
Comment (DanPaseltiner) [reply docs/implementation-guide.md]: Good point!

**Pull request merged**